torch_xla/csrc/tensor_methods.cpp

```diff
@@ -2337,6 +2337,11 @@ XLATensor XLATensor::select(const XLATensor& input, xla::int64 dim,
   return tensor_ops::Select(input, dim, index);
 }
 
+void XLATensor::silu_out(XLATensor& input, XLATensor& out){
+  auto value = input.GetIrValue() * ir::ops::Sigmoid(input.GetIrValue());
```

Comment:
While this will work, I think it is better to define an `Silu` op in `torch_xla/csrc/ops/ops.cpp` and `.h` like
https://github.com/pytorch/xla/blob/403919d167efc64e6d86b269fdd1e2c4a69903de/torch_xla/csrc/ops/ops.cpp#L209

The advantage of doing it this way is when we dump the ir_graph, we will see the op with name `Silu` instead of a `mul` + `sigmoid`. It is easier for debug and it is our convention.